test/pasta-curves-tests.t.sol

```diff
@@ -977,4 +898,37 @@ contract PastaCurvesContractTests is Test {
         assertEq(addition1.y, addition2.y);
         assertEq(addition1.z, addition2.z);
     }
+
```

Comment:
This is great for a start, but from experience, it will very important to have a test for this on the infinity point, both on Pallas and Vesta. This code path (decoding an infinity point) is exercised in practice in nova.